packages/rest/src/utils/url-generator.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ServerObject} from '@loopback/openapi-spec-types';
+
+/**
+ * Describe the options for generating an OpenAPI 3.0.0 standard server url
+ */
+export interface ServerOptions {
+  basePath?: string;
+  description?: string;
+  hostname?: string;
+  protocal?: string;
+  port?: number;
+}
+
+/**
+ * Function to generate an OpenAPI 3.0.0 standard server object
+ * as a string url.
+ * @param serverOptions The options that match interface `ServerOptions`
+ */
+export function createServerAsUrl(serverOptions: ServerOptions): ServerObject {
```

Comment:
As I commented above, LoopBack should not create server entries automatically, therefore this helper should not be need at all and the entire file should be removed.